test/system/src/test/java/io/pravega/test/system/AbstractScaleTests.java

```diff
@@ -35,9 +36,11 @@
     @Getter(lazy = true)
     private final ConnectionFactory connectionFactory = new ConnectionFactoryImpl(false);
     @Getter(lazy = true)
-    private final ClientFactory clientFactory = new ClientFactoryImpl(SCOPE, new ControllerImpl(getControllerURI()));
+    private final ClientFactory clientFactory = new ClientFactoryImpl(SCOPE, new ControllerImpl(getControllerURI(),
+            ControllerImplConfig.builder().retryAttempts(1).build(), getConnectionFactory().getInternalExecutor()));
     @Getter(lazy = true)
-    private final ControllerImpl controller = new ControllerImpl(getControllerURI());
+    private final ControllerImpl controller = new ControllerImpl(getControllerURI(),
+            ControllerImplConfig.builder().retryAttempts(1).build(), getConnectionFactory().getInternalExecutor());
```

Comment:
This implies, the test should not use controller once the clientFactory is closed, since we use the same executor.